samples/wcfapp/Dockerfile.client

```diff
@@ -0,0 +1,8 @@
+FROM microsoft/dotnet-framework:4.7.2-sdk AS build
+WORKDIR /app
```

Comment:
The nested /app/WcfClient/ path seems unnecessary in the context of Docker.  What do you think about eliminating the nesting?  It could be simplified to just `/app` or `/WcfClient`.